### PR TITLE
EZP-25552 (Step 1): UDW Finder View

### PR DESCRIPTION
**Pull request by StephaneDiot**
jira: https://jira.ez.no/browse/EZP-25552

## Description

First step of the UDW explorer : define a finder view having a  selected view.

## Tests

Units and manual tests

## Closed in favor of https://github.com/ezsystems/PlatformUIBundle/pull/722

### CODE REVIEW

**Comment by StephaneDiot** [comment]
ready for review @dpobel @yannickroger

File: Resources/public/js/views/ez-universaldiscoveryview.js [hunk 1]
```diff
@@ -559,7 +559,7 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                     return [
                         new Y.eZ.UniversalDiscoveryBrowseView({
                             bubbleTargets: this,
-                            priority: 100,
+                            priority: 1000,
```
Comment (dpobel) [inline Resources/public/js/views/ez-universaldiscoveryview.js]: why do you change this priority ? Actually, you should completely remove UniversalDiscoveryBrowseView from the list (and adds a deprecation warning when the corresponding module is loaded)

File: Resources/public/js/views/ez-universaldiscoveryview.js [hunk 2]
```diff
@@ -573,6 +573,14 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                             isAlreadySelected: Y.bind(this._isAlreadySelected, this),
                             startingLocationId: this.get('startingLocationId')
                         }),
+                        new Y.eZ.UniversalDiscoveryFinderView({
+                            bubbleTargets: this,
+                            priority: 100,
```
Comment (dpobel) [inline Resources/public/js/views/ez-universaldiscoveryview.js]: you should keep the method list sorted by priority

**Comment by StephaneDiot** [comment]
closed in favor of https://github.com/ezsystems/PlatformUIBundle/pull/722

**Pull request closed without merging**